frontend/src/types/IstioObjects.ts

```diff
@@ -783,6 +783,7 @@ export interface Listener {
   name: string;
   port: number;
   protocol: string;
+  tls?: K8sGatewayTLS | null;
```

Comment:
@ferhoyos it seems null is odd here?

Reply:
yes, now that it is optional you can remove the null here